quickjs.c

```diff
@@ -42193,14 +42193,29 @@ static JSValue js_math_random(JSContext *ctx, JSValueConst this_val,
     return __JS_NewFloat64(ctx, u.d - 1.0);
 }
 
+static double qjs_ceil(double x)
+{
+    return ceil(x);
+}
+
+static double qjs_floor(double x)
+{
+    return floor(x);
+}
+
+static double qjs_sqrt(double x)
+{
+    return sqrt(x);
+}
+
 static const JSCFunctionListEntry js_math_funcs[] = {
     JS_CFUNC_MAGIC_DEF("min", 2, js_math_min_max, 0 ),
     JS_CFUNC_MAGIC_DEF("max", 2, js_math_min_max, 1 ),
     JS_CFUNC_SPECIAL_DEF("abs", 1, f_f, fabs ),
-    JS_CFUNC_SPECIAL_DEF("floor", 1, f_f, floor ),
```

Comment:
Sorry just noticed one thing that I missed last time :sweat: Is there a reason `floor()` and `ceil()` are wrapped but not `fabs()`, `acos()`, etc.?

Reply:
the compiler didnt complained about those.. for some unknown reason